app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -313,8 +314,12 @@ public void partialCallback(ArrayList partialResult) {
 
     @Override
     public void rmsCallback(float rmsdB) {
-        radius = 180 + (int) rmsdB * 2; // 180 is the initial radius
         cParams = listenButton.getLayoutParams();
+        if (initalParams == null){
+            initalParams = cParams.width;
```

Comment:
could get rid of the `{` part and make it one line, but looks good to commit (have not tested it though.)